src/collection/views/CollectionDetail.tsx

```diff
@@ -69,6 +70,9 @@ const CollectionDetail: FunctionComponent<CollectionDetailProps> = ({
 	const [isOptionsMenuOpen, setIsOptionsMenuOpen] = useState<boolean>(false);
 	const [isDeleteModalOpen, setIsDeleteModalOpen] = useState<boolean>(false);
 	const [triggerCollectionDelete] = useMutation(DELETE_COLLECTION);
+	const [relatedCollections, setRelatedCollections] = useState<Avo.Search.ResultItem[] | null>(
```

Comment:
Is het niet meer toepasselijk om deze te te defaulten op [] en dus niet `| null` te typen. Dan heb je geen extra type en kun je gerust nog checken op length in onderstaande if-statement.